packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.2.15
+
+* Reverts changelog change from commit revert.
```

Comment:
4.2.15 is a published (even if retracted) version, so this should have the actual changelog that was in 4.2.15, but with `[Retracted] ` at the start.